### DISCUSSION TITLE
Yetiforce 7 Portal

### DISCUSSION THREAD

**Discussion by FranziskaSom** [YetiForceCompany/YetiForceCRM#17476] [General] [answered]
hi :)
**two questions:**
1. is the yetiforce 7 still compatible with the portal2 because we have difficulties integrating it right now (we also wrote to you/yetiforce on hello@ about this) '...
2. are there long-term plans from yetiforce's side to do without the portal and to solve it via the roles/rights to give the customers access to the crm?

best ...

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
We already have a new v3 portal, which is already written in new react technologies, etc. When we prepare the materials, we will publish more information.

**Reply by FranziskaSom** [reply]
thanks :)

perhaps a rough time frame? we would be happy to test it in advance and in real time. see email to hello from @co***er.de

**Reply by mariuszkrzaczkowski** [reply]
I don't have it planned yet, write to our general email
